### PR TITLE
Press section

### LINKED ISSUE #82: Feature Press Articles
Dahianna & leads to align press highlights (articles and multimedia) and incorporate on page. Similar to the usds.gov "USDS in the news" section on [their homepage](https://www.usds.gov/)

### PR DESCRIPTION
**Pull request by jaredbain**
Create press section on home page

Fixes #82 

## Checklist

- [x] My code follows the existing style, code structure, and naming taxonomy
- [x] I have performed a self-review of my own code and have no open issues
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or syntax errors
- [x] I have verified Draft on actual mobile devices, especially Safari-based

### CODE REVIEW

File: _includes/press.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  <div class="carousel-container snap-inline">
+    <div class="carousel-inner">
+      <div class="track">
+        
+          {% assign sorted = site.articles | sort: 'pageOrderID' | reverse %}
+          {% assign aria_id = press.title | url_encode | truncate: 15, "" %}
+          {% for article in sorted %}
+          <div class="card-container">
+          <div class="press-card {{article.type}}">
+            <a href="{{ article.external_url }}" target="_blank" aria-labelledby="{{ aria_id }}">
```
[the remark this answers is not in the document]
Comment (jaredbain) [reply _includes/press.html]: added recommended code

File: _assets/css/pages/_home.scss [hunk 2]
```diff
@@ -150,3 +150,235 @@
...
+      @include u-padding-bottom(2);
+    }
+    .img-size{
+      width: 400px;
+      height: 230px;
+    }
+    
+  }
+}
+.Press {
```
Comment (DaveSkender) [inline _assets/css/pages/_home.scss:298]: There's probably some [minor] opportunities to refactor in these styles to break out common properties, so you don't have to repeat yourself.  Not super important.

File: _config.yml [hunk 3]
```diff
@@ -157,6 +160,7 @@ assets:
     - _assets/images/logo
     - _assets/images/social
     - _assets/images/stock
+    - _assets/images/articles
```
Comment (DaveSkender) [inline _config.yml:163]: [minor] might be good to separate article images from the left/right nav images, for clarity.

**Review by DaveSkender — COMMENTED** [review]
Generally looks pretty solid.  Just a few thing to potentially explore:

image

Might want to look at some options to collapse the menu sooner or to use smallest words in smaller screens for menu items.  We're running out of space.  This only happens around 1100px wide.

Same applies to the breakpoints for the cards here.  Three cards run past the normal page margin in this same window size.

image

File: _includes/press.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
+
+<section>
+  <div class="carousel-container snap-inline">
+    <div class="carousel-inner">
+      <div class="track">
+        
+          {% assign sorted = site.articles | sort: 'pageOrderID' | reverse %}
```
Comment (DaveSkender) [inline _includes/press.html:7]: date might be a more logical sort data type than manually set numbers, re: `pageOrderID`

File: _config.yml (hunk 3 above)
Comment (jaredbain) [reply _config.yml:163]: Done

File: _includes/press.html (hunk 4 above)
Comment (jaredbain) [reply _includes/press.html:7]: It initially was set to order by date, however the team decided not to sort by date but by article importance

File: _assets/css/pages/_home.scss (hunk 2 above)
Comment (jaredbain) [reply _assets/css/pages/_home.scss:304]: Fixed

**Comment by jaredbain** [comment]
@DaveSkender Made all the changes and just need approval now

**Review by DaveSkender — COMMENTED** [review]
On a smaller screen, I’m never quite able to get a whole card visible when paging, as if the page move is not calibrated to card size.

36C2DBF6-F9C9-474A-A95E-19AFEE970705

**Comment by jaredbain** [comment]
@DaveSkender what screen size is that

**Comment by jaredbain** [comment]
@DaveSkender tablet view has been adjusted

**Comment by jaredbain** [comment]
@DaveSkender Hey Dave, if everything looks good, you can approve the PR

**Review by DaveSkender — APPROVED** [review]
I think there's a few opportunities to make this work a bit better in mobile view (e.g. page increments, photo distortion) and perhaps to refactors some code, but otherwise looks good.

**Pull request merged**